### PR TITLE
[libflac] Update libflac to 1.3.3

### PR DESCRIPTION
**Pull request by janisozaur**

### CODE REVIEW

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: ports/libflac/portfile.cmake [hunk 1]
```diff
@@ -1,47 +1,41 @@
...
-if(VCPKG_LIBRARY_LINKAGE STREQUAL static)
-    file(APPEND ${CURRENT_PACKAGES_DIR}/include/FLAC/export.h "#undef FLAC_API\n#define FLAC_API\n")
-    file(APPEND ${CURRENT_PACKAGES_DIR}/include/FLAC++/export.h "#undef FLACPP_API\n#define FLACPP_API\n")
-endif()
-
-# This license (BSD) is relevant only for library - if someone would want to install
-# FLAC cmd line tools as well additional license (GPL) should be included
-file(COPY ${SOURCE_PATH}/COPYING.Xiph DESTINATION ${CURRENT_PACKAGES_DIR}/share/libflac)
-file(RENAME ${CURRENT_PACKAGES_DIR}/share/libflac/COPYING.Xiph ${CURRENT_PACKAGES_DIR}/share/libflac/copyright)
+include(vcpkg_common_functions)
```
[the remark this answers is not in the document]
Comment (janisozaur) [reply ports/libflac/portfile.cmake]: Done

**Review by JackBoosY — APPROVED** [review]

**Comment by ras0219-msft** [comment]
LGTM, thanks for the update!

**Pull request merged**